src/xunit.performance.api/XunitPerformanceHarness.cs

```diff
@@ -68,7 +68,9 @@ public void RunBenchmarks(string assemblyFileName)
 
             void xUnitAction(string assemblyPath)
             {
-                XunitRunner.Run(assemblyPath, _typeNames);
+                var errCode = XunitRunner.Run(assemblyPath, _typeNames);
+                if (errCode != 0)
+                    throw new Exception($"{errCode} benchmark(s) did not complete successfully.");
```

Comment:
I'd recommend making this text a bit more clear.  I think you're trying to say that some number of benchmarks actually failed to execute (perhaps threw an exception).  We should try to make clear that these tests failed to execute - not that the performance of the tests was not acceptable.